converter/index.js

```diff
@@ -1,15 +1,67 @@
-var momentTimezoneJson = require("./latest.json");
+const packedLatest = require('moment-timezone/data/packed/latest.json');
+const moment = require('moment-timezone');
+// const oldLatest = require('./latest.json');
 
-var output = {
-  version: momentTimezoneJson.version,
-  zoneData: {}
-};
+function unpackAndUnlink(latest) {
+  const unpacked = {
+    ...latest,
+    zones: latest.zones.map(moment.tz.unpack),
+    links: [],
+    countries: []
+  };
 
-momentTimezoneJson.zones.forEach(({name, abbrs, untils, offsets, isdsts}) => {
-  name = name.split('/');
-  name.slice(0, -1).reduce(function (tree, item) {
-    return tree[item] || (tree[item] = {});
-  }, output.zoneData)[name.slice(-1)[0]] = {abbrs, untils, offsets, isdsts};
-})
+  latest.links.forEach(link => {
+    const [ leadTzName, alias ] = link.split('|');
+    const leadTzData = unpacked.zones.find(zone => zone.name === leadTzName);
 
-console.log(JSON.stringify(output, null, 2));
+    unpacked.zones.push({
+      ...leadTzData,
+      name: alias
+    });
+  });
+
+  latest.countries.forEach((country, i) => {
+    const [ countryAlias, leadTzName ] = country.split('|');
+    const leadTzData = unpacked.zones.filter(zone => leadTzName.includes(zone.name));
+    leadTzData.forEach(lead => {
+      lead.countries = lead.countries ? [...lead.countries, countryAlias] : [countryAlias];
+    })
+    unpacked.countries.push({
+      name: countryAlias,
+      zones: leadTzName.split(" ")
+    });
+  });
+
+  const sortFn = (a, b) => {
+    return a.replace('/', '').toLowerCase().localeCompare(b.replace('/', '').toLowerCase())
+  }
+
+    unpacked.zones = [
```

Comment:
This is still to help me understand if the data was the same, as the difference in sorting or where the objects were located were making it difficult in a huge file. We can discard this if we move on with this solution